content/documentation/install-uninstall.md

```diff
@@ -24,6 +24,16 @@ If you’re eager to get started, this will install the complete Quire package b
 
 The Quire installation process may take a minute or two, during which time there will be messaging output in Terminal. The only messages of any concern are those labeled as ERROR or ERR. If you see these errors, or if you see "command not found" after entering `quire --help` in step 4., search our [Discussions Forum](https://github.com/thegetty/quire/discussions) to troubleshoot installation issues.
 
+
+{{< q-class "box warning tip" >}}
+- Don't fret if you see see an error (ERR!) message in your Terminal that reads: "permission denied." Type the following command and hit enter:
+```
+cd
+sudo chown -R $USER /usr/local
+```
+You'll be asked to type your computer password (it will be invisible) and will most likely receive an error message reading “operation not permitted” or "operation denied". Ignore this! Go back to \#3. and run the install command again.
```

Comment:
This surprises me. If it says “operation not permitted” or "operation denied", then nothing should have changed by running the command and so the command shouldn't have any effect on fixing the install process.

If the install working for people after they see “operation not permitted” or "operation denied", it must only be because they ran the install process twice.

Reply:
Holding off on this PR until we can test running install process twice to fix the issue v. `sudo chown`

Reply:
@geealbers Revised this tip with the info provided by Daniel Keller. Is it worth it to go into the "administrator" bit?

Reply:
I don't necessarily think the administrator stuff is all needed. But I also think this edit isn't quite right. You don't need the `cd` in the command here because they should already be in their home directory. Also, this warning/tip is directly related to the paragraph right above it. I'd suggest combining the paragraph and the tip. Something like:

> The Quire installation process may take a minute or two, during which time there will be messaging output in Terminal. The only messages of any concern are those labeled as ERROR or ERR. In particular, if you see error messages that say "permission denied" you will need to run the install command as an administrator or "super user". You can do this with the `sudo` ("superuser do") command: 
> 
> ```
> sudo npm install --global @thegetty/quire-cli
> ```
> 
> You will be prompted to enter your password. If after running the command with `sudo` the permission errors are not resolved, or if you see other errors, or get a "command not found" message after entering `quire --help` in step 4., search our Discussions Forum to troubleshoot installation issues.